umm-spec-lib/test/cmr/umm_spec/test/migration/version_migration.clj

```diff
@@ -1186,7 +1186,20 @@
   (is (= "COMPLETE"
          (:CollectionProgress
            (vm/migrate-umm {} :collection "1.9" "1.10"
-                          {:CollectionProgress "COMPLETE"}))))))
+                          {:CollectionProgress "COMPLETE"}))))
+  (testing "VerticalSpatialDomains migration from 1.9.0 to 1.10.0"
+    (let [vsds {:VerticalSpatialDomains [{:Type "An Invalid Type"
```

Comment:
What happens if we have one with a type "AtmosphereLayer"?

Reply:
Added a test and it passes